src/ui/pages/payment-entry-page.svelte

```diff
@@ -48,6 +48,8 @@
   import StripeElementsComponent from "../molecules/stripe-elements.svelte";
   import PriceUpdateInfo from "../molecules/price-update-info.svelte";
 
+  type View = "loading" | "form" | "message";
```

Comment:
```suggestion
  type View = "loading" | "form" | "error";
```

instead? but no big deal

Reply:
I like it, on it.